code/game/objects/items/weapons/material/misc.dm

```diff
@@ -71,14 +71,16 @@
 	w_class = ITEMSIZE_SMALL
 	attack_verb = list("mushed", "splatted", "splooshed", "splushed") // Words that totally exist.
 
-/obj/item/weapon/material/snow/snowball/attack_self(mob/user as mob)
+/obj/item/weapon/material/snow/snowball/attack_self(mob/user as mob)	//TFF 22/33/20 - changes to to_chat line to be show only to self. ToDo: proper fix for the thing.

```

Comment:
???

Reply:
Ehhhh, I s'pose I could remove that considering that the visible_message isn't working right there. Wanted to have a reminder to return to it and figure out _why_ it hasn't been working the way it should.